### DISCUSSION TITLE
Fiyat ne olabilir sizce gençler tahminler etkileşim olsun

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#114] [Q&A] [answered]
Fiyat ne olabilir

**Comment by cartvecurt** [comment]
Şimdiden tahmin yapmak güç ama uzun vadede 50 dolarlar seviyesini görebilir gibi düşünüyorum.

**Reply by rocknstorm** [reply]
Gerçekten o kadar olur mu?

**Reply by Akrepkapo** [reply]
Olmaz kankam okadar

**Reply by cartvecurt** [reply]
Adamlar blockchain ekosistemi kuruyorlar normal bir proje değil ki, eğer işler istedikleri gibi giderse 50den çok daha yukarısı mümkün

**Reply by ByWhite20** [reply]
100 bile olur

**Reply by hakann68** [reply]
bence 0.20 cent ten başlar

**Reply by brkymstf** [reply]
Hedefleri USDT bazında bir fiyatlandırma mı? Yoksa ekosistemler arasında bir yere mi sahip olmak... Bunun ile ilgili bir bilgi var mı? Ağ hızı - hareketliliği - blocklararası işlem güvenirliği gibi bir çok konu fiyatı belirler... umarım güzel fiyatlar olur da hepimizin kasası yeşillenir... tahminim giriş aşamasında 0.7-0.9 usd...

**Comment by byblckd** [comment, marked as answer]
Max arz 2.1 milyar adet coin yakım yapmadığını düşünerek hareket edersek 1dolar olması 2.1 milyar dolar piyasa değeri demek uzun vadede fiyatı yapacağı projeler ile kullanılan bir blockchain sisteminin ilgi çekmesi ve tercih edilmesiyle mümkün olabilir. Fiyatı kafanızda canlandırabilmek için toplam arzın oluşturacağı piyasa değerini gözünüzde canlandırın yoksa shiba sahiplerinin 1 dolar olacak shiba demesi gibi gülünç haller ile karşı karşı ya kalırsınız

**Reply by cartvecurt** [reply]
Uzun vade dememin nedeni o zaten, yüksek bir açılış olacağını sanmıyorum ben de

**Reply by byblckd** [reply]
tabiki de kripto piyasasının toplam değeri artıkça hedef yükseltilebilir ama çoğu insan başlangoçtaki düüşük değerleri görünce bir daha yüzüne bakmayacak ama sabırlı olanlar uzun vade de güzel kar alabilirler herşey proje sahiplerinin elinde

**Reply by cartvecurt** [reply]
Katılıyorum, böyle işlerde her zaman uzun vade önemli

**Comment by hemresenol** [comment]
Uzun vadede 100 doları görür yatırım tavsiyesidir 😁
